ios/FluentUI/Bottom Commanding/BottomCommandingController.swift

```diff
@@ -445,9 +446,7 @@ open class BottomCommandingController: UIViewController {
             static let heroStackTopBottomMargin: CGFloat = 16
 
             static let moreButtonIcon: UIImage? = UIImage.staticImageNamed("more-24x24")
-
-            // TODO: Localize
-            static let moreButtonTitle: String = "More"
+            static let moreButtonTitle: String = "CommandingBottomBar.More".localized
```

Comment:
not sure the more icon image and the title be in the constant struct. shouldn't these be part of runtime code? not sure what binary size impact is for swift code even though it might be small...

Reply:
Is there something specific about static lets that increases binary size? The String.localized getter and the UIImage constructor should are executed lazily anyways, because static lets are lazy by default.

Reply:
@rdeassis